packages/react-ui-components/src/CheckBox/style.css

```diff
@@ -25,7 +25,6 @@
     border: 1px solid var(--brandColorsContrastBrighter);
     vertical-align: text-bottom;
     background: var(--brandColorsContrastDark);
-    color: var(--brandColorsContrastBrightest);
     font-size: 20px;
     font-weight: bold;
     color: #5b5b5b;
```

Comment:
Btw, what's this color 5b5b5b? Haven't seen it anywhere before.
If it's really needed I guess we should add it to color constants, no?

Reply:
It's color of the checkbox icon of the old ui. I'll add it to the color constants.

Reply:
Does it look really different from `brandColorsContrastBrightest` or any of other colors that we have? Maybe we can leave it normalize? Don't know, just asking.

Reply:
![bright](https://user-images.githubusercontent.com/12071529/31883562-8c19214e-b7ea-11e7-95da-d7151f5d112e.png)

This would be bright. The other ones wouldn't fit. 
Do you think this is ok? Otherwise we need to merge this: https://github.com/neos/brand/pull/9

Reply:
Anyway, @crazydrecksack merged my color PR, so I will use this.